stripe-core/src/main/java/com/stripe/android/core/exception/ExceptionUtils.kt

```diff
@@ -0,0 +1,12 @@
+@file:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+
+package com.stripe.android.core.exception
+
+import androidx.annotation.RestrictTo
+
+@get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+val Throwable.safeAnalyticsMessage: String
```

Comment:
Nice idea! I went with the custom `analyticsValue()` to provide the same values like on iOS, but now I wish I had used the class name from the start. Would it make sense to make `analyticsValue()` an open method that provides the class name by default, but can be overridden?